app/deepseg.cc

```diff
@@ -631,11 +663,15 @@ int main(int argc, char* argv[]) try {
 		// copy new frame to buffer
 		cap.retrieve(raw);
 		ti.retrns = timestamp();
-		ai.set_input_frame(raw);
 		ti.copyns = timestamp();
```

Comment:
This line should stay with the previous one. Just move it down to after line 673 too …

Reply:
Actually the code is:

```
		// copy new frame to buffer
		cap.retrieve(raw);
		ti.retrns = timestamp();

		if (raw.rows == 0 || raw.cols == 0) continue; // sanity check

		if ( crop_region.height) {
			raw(crop_region).copyTo(raw);
		}
		ai.set_input_frame(raw);
		ti.copyns = timestamp();

		// do background detection magic
```
I hope that moving the times are set correctl.